core/string/translation_server.h

```diff
@@ -52,10 +52,14 @@ class TranslationServer : public Object {
 
 	static inline TranslationServer *singleton = nullptr;
 	bool _load_translations(const String &p_from);
-	String _standardize_locale(const String &p_locale, bool p_add_defaults) const;
 
 	static void _bind_methods();
 
+#ifndef DISABLE_DEPRECATED
+	String _standardize_locale_compat_98972(const String &p_locale) const;
```

Comment:
```suggestion
	String _standardize_locale_bind_compat_98972(const String &p_locale) const;
```
Standard naming

Reply:
Whoops. I thought I read the compatibility guide correctly :stuck_out_tongue_closed_eyes: Good catch.